main/createUiDefinition.json

```diff
@@ -40,21 +40,9 @@
         "label": "VM size",
         "toolTip": "Select the size of VM.",
         "recommendedSizes": [
+          "Standard_F8s_v2",
```

Comment:
can we add more than one in recommended sizes ?

Reply:
Yes, we can add more than one recommended sizes, if the first one is somehow not available then deployment will take the next one and so on.